scripts/process_vcs.py

```diff
@@ -64,8 +64,9 @@ def vcs_download(obsid, start_time, stop_time, increment, data_dir,
     logger.info("Downloading files from archive")
     voltdownload = "voltdownload.py"
     obsinfo = meta.getmeta(service='obs', params={'obs_id':str(obsid)})
+    comb_del_check = meta.combined_deleted_check(obsid, begin=start_time, end=stop_time)
     data_format = obsinfo['dataquality']
-    if data_format == 1:
+    if data_format == 1 or (comb_del_check and data_format == 6):
```

Comment:
A comment here might be handy because what this is checking isn't obvious

Reply:
Done